gen/definitions/url_group.yaml

```diff
@@ -0,0 +1,43 @@
+---
+name: Url Group
+rest_endpoint: /api/fmc_config/v1/domain/{DOMAIN_UUID}/object/urlgroups
+data_source_name_query: true
+doc_category: Objects
+attributes:
+  - model_name: name
+    type: String
+    description: User-created name of the resource.
```

Comment:
Mentioning "resource" might be misleading.. as it is not the name of the TF resource, but rather the object it creates. So maybe something like "User-created name of the object.".

Reply:
Sure, makes sense